pkg/aws/core.go

```diff
@@ -171,6 +176,38 @@ func (d *Driver) CreateMachine(ctx context.Context, req *driver.CreateMachineReq
 	return response, nil
 }
 
+//returns Placement Object required in ec2.RunInstancesInput
+func getPlacementObj(req *driver.CreateMachineRequest) *ec2.Placement {
+
+	tenancy := "default"
+	affinity := "default"
+	var availZone string
+
+	myannotations := req.Machine.Spec.NodeTemplateSpec.ObjectMeta.Annotations
```

Comment:
Nit picking - How about using a better variable name instead of `myannotations`?